File: Core/Entities/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Core.Entities
{
    public class Booking : BaseEntity
    {
        public int UserId { get; set; }
        public int ChargingPointId { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public decimal Price { get; set; }
        public DateTime CreatedTimestamp { get; set; }
    }
}

File: Client/src/app/shared/models/booking.ts
export interface IBooking {
  id: number;
  userId: string;
  chargingPointId: number;
  start: string;
  end: string;
  price: number;
  createdTimestamp: string;
}

export interface IBookingToCreate {
  id: number;
  UserId: string;
  ChargingPointId: number;
  Start: string;
  End: string;
  Price: number;
  CreatedTimestamp: string;
}
